$OpenBSD: patch-Makefile,v 1.3 2020/06/17 15:24:30 thfr Exp $

set the installdir to PREFIX/share rather than PREFIX/lib
hardcode VERSION, instead of getting it from git
don't install IP2LOCATION-LITE-DB1.IPV6.BIN.ZIP; use libmaxminddb from
ports instead

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -58,7 +58,7 @@ datadir ?= $(datarootdir)
 mandir ?= $(datarootdir)/man/
 bindir ?= $(prefix)/bin
 libdir ?= $(prefix)/lib
-gameinstalldir ?= $(libdir)/openra
+gameinstalldir ?= $(datarootdir)/openra
 
 BIN_INSTALL_DIR = $(DESTDIR)$(bindir)
 DATA_INSTALL_DIR = $(DESTDIR)$(gameinstalldir)
@@ -76,13 +76,13 @@ INSTALL_PROGRAM = $(INSTALL) -m755
 INSTALL_DATA = $(INSTALL) -m644
 
 # Toolchain
-MSBUILD = msbuild -verbosity:m -nologo
+MSBUILD = msbuild -verbosity:d -nologo -clp:DisableConsoleColor
 
 # Enable 32 bit builds while generating the windows installer
 WIN32 = false
 
 # program targets
-VERSION     = $(shell git name-rev --name-only --tags --no-undefined HEAD 2>/dev/null || echo git-`git rev-parse --short HEAD`)
+VERSION     = release-20200503
 
 # dependencies
 UNAME_S := $(shell uname -s)
@@ -101,7 +101,7 @@ check-scripts:
 check: dependencies
 	@echo
 	@echo "Compiling in debug mode..."
-	@$(MSBUILD) -t:build -p:Configuration=Debug
+	@$(MSBUILD)
 	@echo
 	@echo "Checking runtime assemblies..."
 	@mono --debug OpenRA.Utility.exe all --check-runtime-assemblies $(WHITELISTED_OPENRA_ASSEMBLIES) $(WHITELISTED_THIRDPARTY_ASSEMBLIES) $(WHITELISTED_CORE_ASSEMBLIES)
@@ -157,7 +157,7 @@ ifeq ($(WIN32), $(filter $(WIN32),true yes y on 1))
 else
 	@$(MSBUILD) -t:build -p:Configuration=Release
 endif
-	@./fetch-geoip.sh
+#	@./fetch-geoip.sh
 
 clean:
 	@ $(MSBUILD) -t:clean
@@ -169,10 +169,8 @@ clean:
 distclean: clean
 
 cli-dependencies:
-	@./thirdparty/fetch-thirdparty-deps.sh
 	@ $(CP_R) thirdparty/download/*.dll .
 	@ $(CP_R) thirdparty/download/*.dll.config .
-	@ test -f OpenRA.Game/obj/project.assets.json || $(MSBUILD) -t:restore
 
 linux-dependencies: cli-dependencies linux-native-dependencies
 
@@ -219,7 +217,6 @@ install-engine:
 	@$(INSTALL_DATA) VERSION "$(DATA_INSTALL_DIR)/VERSION"
 	@$(INSTALL_DATA) AUTHORS "$(DATA_INSTALL_DIR)/AUTHORS"
 	@$(INSTALL_DATA) COPYING "$(DATA_INSTALL_DIR)/COPYING"
-	@$(INSTALL_DATA) IP2LOCATION-LITE-DB1.IPV6.BIN.ZIP "$(DATA_INSTALL_DIR)/IP2LOCATION-LITE-DB1.IPV6.BIN.ZIP"
 
 	@$(CP_R) glsl "$(DATA_INSTALL_DIR)"
 	@$(CP_R) lua "$(DATA_INSTALL_DIR)"
